pkg/generators/forwarding/fluentd/generators.go

```diff
@@ -196,14 +197,26 @@ func (engine *ConfigGenerator) generateSourceToPipelineLabels(sourcesToPipelines
 }
 
 func (engine *ConfigGenerator) generatePipelineToOutputLabels(pipelines []logging.PipelineSpec) ([]string, error) {
-	configs := []string{}
+	var configs []string
 	for _, pipeline := range pipelines {
+		var jsnLabels string
+
+		if pipeline.Labels != nil {
+			jl, err := json.Marshal(pipeline.Labels)
+			if err != nil {
+				return nil, err
```

Comment:
Wrapping the error with a nice message `fmt.Errorf("failed marshaling pipelines labels: %s", err)` would make the error side easy to identify.

Reply:
Fixed